### PR TITLE
ticdc: add ga version, remove experiment warning

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

TiCDC will GA since v4.0.5

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x]  v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @overvenus @TomShawn

**Review by leoppro — COMMENTED** [review]
LGTM

**Review by overvenus — APPROVED** [review]
LGTM 🎉

File: ticdc/ticdc-overview.md [hunk 1]
```diff
@@ -5,9 +5,7 @@ aliases: ['/docs-cn/dev/ticdc/ticdc-overview/','/docs-cn/dev/reference/tools/tic
 
 # TiCDC 简介
 
-> **警告：**
->
-> TiCDC 目前为实验特性，不建议在生产环境中使用。
+> TiCDC 从 v4.0.5 正式 GA，生产环境推荐使用 >= v4.0.5 的版本
```
Comment (TomShawn) [inline ticdc/ticdc-overview.md:8]:
```suggestion
> **注意：**
>
> TiCDC 从 v4.0.5 起成为正式功能，可用于生产环境。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**